src/Entities/Player.h

```diff
@@ -528,6 +528,9 @@ class cPlayer :
 	The player removes its m_ClientHandle ownership so that the ClientHandle gets deleted. */
 	void RemoveClientHandle(void);
 
+	/** Returns the relative block hardness for the block a_Block */
```

Comment:
Relative to what? What stats does this take into account? (block type, player surroundings, equipped tool...)